docker/ubuntu16.04/Dockerfile

```diff
@@ -0,0 +1,378 @@
+# This file creates docker containers of the G+Smo library
+#
+# G+Smo (pronounced gismo or gizmo) is a C++ library for isogeometric
+# analysis (IGA). Geometry plus simulation modules aims at the
+# seamless integration of Computer-aided Design (CAD) and Finite
+# Element Analysis (FEA).
+#
+# http://gs.jku.at/gismo
```

Comment:
please change link to https://github.com/gismo

Reply:
(same in other files)

Reply:
Done